content/en/docs/File Format/implementationstatus.md

```diff
@@ -0,0 +1,101 @@
+| UNKNOWN                                   |       |        |       |       |
+
+### Encoding
+
+| Encoding                                  | C++   | Java   | Go    | Rust  |
+| ----------------------------------------- | ----- | ------ | ----- | ----- |
+| PLAIN                                     |       |        |       |       |
+| PLAIN_DICTIONARY                          |       |        |       |       |
+| RLE_DICTIONARY                            |       |        |       |       |
+| RLE                                       |       |        |       |       |
+| BIT_PACKED                                |       |        |       |       |
+| DELTA_BINARY_PACKED                       |       |        |       |       |
+| DELTA_LENGTH_BYTE_ARRAY                   |       |        |       |       |
+| DELTA_BYTE_ARRAY                          |       |        |       |       |
+| BYTE_STREAM_SPLIT                         |       |        |       |       |
+
+### Compression
+
+| Compression                               | C++   | Java   | Go    | Rust  |
+| ----------------------------------------- | ----- | ------ | ----- | ----- |
+| UNCOMPRESSED                              |       |        |       |       |
+| SNAPPY                                    |       |        |       |       |
+| GZIP                                      |       |        |       |       |
+| LZO                                       |       |        |       |       |
+| BROTLI                                    |       |        |       |       |
+| LZ4                                       |       |        |       |       |
+| ZSTD                                      |       |        |       |       |
+| LZ4_RAW                                   |       |        |       |       |
+
+### Other format level features
+
+|                                           | C++   | Java   | Go    | Rust  |
+| ----------------------------------------- | ----- | ------ | ----- | ----- |
+| xxHash Bloom filters                      |       |        |       |       |
+| bloom filter length                       |       |        |       |       |
+| Statistics min_value, max_value           |       |        |       |       |
+| Column index                              |       |        |       |       |
+| Offset index                              |       |        |       |       |
+| Modular encryption                        |       |        |       |       |
+| Page CRC32 checksum                       |       |        |       |       |
+| Modular encryption                        |       |        |       |       |
+
+### High level data API-s for parquet feature usage
+
+| Format                                       | C++   | Java   | Go    | Rust  |
+| -------------------------------------------- | ----- | ------ | ----- | ----- |
+| Hive-style partitioning                      |       |        |       |       |
```

Comment:
I don't think this is something that is generally part of parquet?

Reply:
It’s not part of the parquet format, on the other hand dataset and catalog features are part of all the implementations. I’m open to skip it in the first round

Reply:
Partitioning really seems irrelevant here, especially as it can be applied to other data types such as CSV etc.

Other features such as pruning using statistics, however, are definitely Parquet-related and could be mentioned.